.github/workflows/danger.yml

```diff
@@ -0,0 +1,17 @@
+name: Danger
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.5.6"
+          bundler-cache: true
+      - run: bundle exec danger
```

Comment:
You'll need a token for it to complain successfully in PRs. Copy from https://github.com/ruby-grape/grape/blob/master/.github/workflows/danger.yml.

Reply:
Updated.